man/systemd.network.xml

```diff
@@ -899,6 +895,47 @@
     </refsect1>
 
   <refsect1>
+    <title>[IPv6AcceptRouterAdvertisements] Section Options</title>
```

Comment:
I'm not convinced that we need a separate section for this. C.f. the [DHCPDUID] conversation.

But if we *do* create a separate section, then it should be called `IPv6RouterAdvertisments`. `IPv6AcceptRouterAdvertisements=` should be deprecated and replaced with a new option in this section, with some shorter name (Accept or AcceptAdvertisements or AcceptRouterAdvertisements, dunno).